packages/de1/src/converters/index.ts

```diff
@@ -22,9 +22,10 @@ export { ShotFrame } from "./shotFrame";
 export { Calibrate } from "./calibrate";
 
 export type ConverterKey = keyof Converters;
-export type ConverterValue<N extends ConverterKey> = Value<Converters, N>;
+export type ConverterValue = Value<Converters>;
 
 export interface Converters {
+  [key: string]: Converter<any>
```

Comment:
Getting this error otherwise:

```
[!] (plugin rpt2) Error: /Users/oscar/dev/priv/de1/packages/de1/src/de1.ts(7,29): semantic error TS2344: Type 'Converters' does not satisfy the constraint 'Record<string, Converter<any>>'.
  Index signature is missing in type 'Converters'
```

Reply:
👌